packages/paste-website/tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "skipLibCheck": true /* Skip type checking all .d.ts files. */,
     "outDir": "dist/",
     "emitDeclarationOnly": false,
-    "typeRoots": ["../../@types", "../../node_modules/@types", "./types"]
+    "typeRoots": ["../../@types", "../../node_modules/@types", "./types"],
+    "noUnusedLocals": false
```

Comment:
This is required for paste-website as the tsconfig has "jsx":  "react-jsx". This is needed because we're using "jsxImportSource": "@emotion/react". This allows us to re-introduce the 'import * as React from 'react';' at the top of the test files without the type-check step failing in the pipeline

Reply:
thought: let's not add `"noUnusedLocals": false`. While it may solve the React import problem that was randomly introduced, it opens up a lot of room for leaving unused locals in all the other code.

Reply:
Okay understood, I'll trace back where the react import originally came from and I'll remove it.